astrocut/cube_cut.py

```diff
@@ -4,23 +4,21 @@
 
 import os
 import warnings
-
-from time import time
 from itertools import product
+from time import time
 
+# Note: Use the astropy function if available, TODO: fix > 4.3 astropy fitting
+import astropy
 import astropy.units as u
 import numpy as np
-
 from astropy import wcs
 from astropy.coordinates import SkyCoord
 from astropy.io import fits
 from astropy.time import Time
 
-from . import __version__ 
+from . import __version__
 from .exceptions import InputWarning, InvalidQueryError
 
-# Note: Use the astropy function if available, TODO: fix > 4.3 astropy fitting
-import astropy
 if astropy.utils.minversion(astropy, "4.0.2") and (float(astropy.__version__[:3]) < 4.3):
```

Comment:
Should probably switch to version.parse at some point here...

from packaging import version

Also, I think we can bump the minimum version of astropy. Pretty soon I think we're going to require >= 5.2 for the cloud stuff, so maybe this can just get dropped in a later PR.

Reply:
I've made a JIRA ticket to address these two comments in Q2.